backend/app/assets/stylesheets/spree/backend/shared/_forms.scss

```diff
@@ -124,6 +124,11 @@ button, .button {
     width: 100%;
     text-align: center;
   }
+
+  &[disabled], &.disabled {
```

Comment:
I think the more preferred way would be to use `:disabled` CSS3 selector. Although they are the same, some explanation why use one over the `[disabled]` can be found here:
http://stackoverflow.com/a/20146936/336806

It works in cross-browser:
http://caniuse.com/#feat=css-sel3